inskinesis/inskinesis.go

```diff
@@ -274,7 +273,7 @@ func (s *stream) Put(record interface{}) {
 
 func (s *stream) putRecords(batch []*kinesis.PutRecordsRequestEntry, retryCount int) (int, error) {
 	fmt.Printf("Sending %d records to Kinesis stream %s\n", len(batch), s.name)
-	if retryCount == 0 {
+	if retryCount == -1 {
```

Comment:
this seems counterintuitive, maybe a better naming of the variable and using `<` or `>` instead of `==` will help